bundle/DependencyInjection/NetgenIbexaScheduledVisibilityExtension.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Bundle\IbexaScheduledVisibilityBundle\DependencyInjection;
+
+use Exception;
+use Symfony\Component\Config\FileLocator;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Loader;
+use Symfony\Component\HttpKernel\DependencyInjection\Extension;
+
+final class NetgenIbexaScheduledVisibilityExtension extends Extension
+{
+    public function getAlias(): string
+    {
+        return 'netgen_ibexa_scheduled_visibility';
+    }
+
+    public function getConfiguration(array $config, ContainerBuilder $container): Configuration
+    {
+        return new Configuration($this->getAlias());
+    }
+
+    /**
+     * @throws Exception
```

Comment:
Not thrown.

Reply:
fixed in fa153b003652a49ea510b3b043c68e87d73d325f